share/spice/forecast/forecast.js

```diff
@@ -119,12 +119,11 @@
     }
 
     var build_daily = function(f) {
-      var dailyObj = [],
-          day_strs = [],
-          today = new Date(),
-          today_i = today.getDay(),
-          month_i = today.getMonth(),
-          date_i = today.getDate(),
+      var dailyObj = [],          
+          today = moment(),
+          today_i = today.day(),
+          month_i = today.month(),
+          date_i = today.date(),
```

Comment:
@jagtalon Good spot. Some of these variables weren't even used before. I've removed them in the latest commit.